components/mothership/makefile

```diff
@@ -0,0 +1,24 @@
+export OAPI_GENERATOR=oapi-codegen
+
+export OAPI_FILE=./docs/external_api.yaml
+
+docs/external_api.yaml:
+	# TODO make sure to fetch oapi from main when it will be available!
+	curl -o $(OAPI_FILE) --create-dirs https://raw.githubusercontent.com/kyma-incubator/reconciler/3a5c93d64b9f1078cf07f7fae8915d75ed20b6e6/openapi/external_api.yaml
```

Comment:
Maybe worth to directly externalise the revision-id to be able to upgrade it in via the CI system (in the future).

E.g. moving it into a separate config.sh file (e.g. like [here](https://github.com/clearlinux/common/blob/master/Makefile.toplevel)) so that it can be updated via simple bash script.